app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
-  def full_title page_title
-    base_title = t "base_title"
-    page_title.blank? ? base_title : page_title + " | " + base_title
+  def full_title page_title = ""
```

Comment:
```suggestion
  def full_title page_title
```
k cần gán giá trị mặc định